website/client/components/userMenu/profile.vue

```diff
@@ -3,18 +3,18 @@ div
   b-modal#profile(title="Profile", size='lg', :hide-footer="true")
     div(slot='modal-header')
       .profile-actions
-        button.btn.btn-secondary(@click='sendMessage()')
+        button.btn.btn-secondary(:title="$t('sendMessage')", @click='sendMessage()', v-b-tooltip.hover.left="")
```

Comment:
so when a :title attribute is defined the tooltip can be emtpy?

Reply:
Hmm I was going off what @negue had suggested to get the tooltip to function properly. Should I put in what the tooltip would output?

Reply:
`v-b-tooltip.hover.left` just can't be written in the syntax, cause its an empty attribute, `:title` is used as the tooltip value

Reply:
Yeah, but cannot it be used like here https://github.com/HabitRPG/habitica/blob/develop/website/client/components/memberDetails.vue#L23 ?

Reply:
oh, I just took the bootstrap-vue docs for it

@xXenithx  could you try if it works the same as in memberDetails? if so, please use that syntax, looks much cleaner^^

Reply:
@paglias @negue Yeah just changed it according to memberDetails and it does in fact work. I'll commit the changes to the PR.